src/templates/admin/journal/submissions.html

```diff
@@ -27,92 +27,32 @@
-                <div class="title-area"><h2>{% trans 'Peer Review' %}</h2></div>
-                <p>{{ journal_settings.general.peer_review_info|safe }}</p>
-                {% endif %}
-                {% if journal_settings.general.publication_cycle %}
-                <div class="title-area"><h2>{% trans 'Publication Cycle' %}</h2></div>
-                <p>{{ journal_settings.general.publication_cycle|safe }}</p>
-                {% endif %}
-                <div class="title-area"><h2>{% trans 'Sections' %}</h2></div>
-                    <div class="row expanded">
-                        <div class="large-6 columns">
-                            <p><strong>Section</strong></p>
-                        </div>
-                        <div class="medium-2 columns">
-                            <p><strong>{% trans "Public Submissions" %}</strong></p>
-                        </div>
-                        <div class="medium-2 columns">
-                            <p><strong>{% trans "Peer Reviewed" %}</strong></p>
-                        </div>
-                        <div class="medium-2 columns">
-                            <p><strong>{% trans "Indexed" %}</strong></p>
-                        </div>
-                    </div>
-                {% for section in sections %}
-                    <div class="row">
-                        <div class="large-6 columns">
-                            <p>{{ section.name }}</p>
-                        </div>
-                        <div class="medium-2 columns">
-                            <strong>{% if section.public_submissions %}<i class="fa fa-check"></i>{% else %}<i class="fa fa-times"></i>{% endif %}</strong>
-                        </div>
-                        <div class="medium-2 columns">
-                            <strong>{% if section.number_of_reviewers > 0 %}<i class="fa fa-check"></i>{% else %}<i class="fa fa-times"></i>{% endif %}</strong>
-                        </div>
-                        <div class="medium-2 columns">
-                            <strong>{% if section.indexing %}<i class="fa fa-check"></i>{% else %}<i class="fa fa-times"></i>{% endif %}</strong>
-                        </div>
-                        
-                    </div>
+                {% for item in submission_items %}
+                    {% if item.title == 'licences' %}
+                        <h3>{% trans 'Licences' %}</h3>
+                        <p>{{ request.journal.name }} {% trans "allows the following licences for submission" %}:</p>
+                        {% include "elements/license_block.html" %}
+                    {% elif item.title == 'sections' %}
+                        {% include "elements/section_block.html" %}
+                    {% elif item.get_display_text %}
+                        <h3>{{ item.title|capfirst }}</h3>
+                        {{ item.get_display_text|safe }}
+                    {% endif %}
+                    <hr />
```

Comment:
This change seems unrelated to this issue, has it leaked from another branch?

Reply:
Yes and no - it updates the submissionly only version of this page to work with the submission items so I included it.